solidity-v1/dashboard/src/connectors/injected-connector.js

```diff
@@ -50,7 +56,7 @@ class InjectedConnector extends AbstractConnector {
   }
 
   disconnect = async () => {
-    // window.ethereum injected by MetaMask does not provide a method to
+    // window.tally injected by Tally does not provide a method to
```

Comment:
This applies to both MetaMask and Tally so we should probably change this comment to something like:

```suggestion
    // window.ethereum injected by MetaMask and window.tally injected by Tally does not provide a method to
```

Reply:
done